packages/core/src/util.ts

```diff
@@ -183,7 +183,7 @@ export function truncate(str: string, len: number) {
 export async function getIdFromEmail(email: string) {
     const id = bytesToHex(
         await getCryptoProvider().hash(
-            stringToBytes(email.trim().toLocaleLowerCase()),
+            stringToBytes((email || "").trim().toLocaleLowerCase()),
```

Comment:
I ran into this error when trying to delete an account. I didn't investigate further if it could/should be stopped before as it was out of scope. I'm happy to look further into it if you think it's a good idea.